recipes/gulp/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "gulp" %}
+{% set version = "3.9.1" %}
+{% set sha256 = "9616c6bc8ca165375832bdc096260931369d5339edef2e0a4d48bc7a05b04d15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/gulpjs/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: 
+    - npm install -g gulp
+ 
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
```

Comment:
Can you please fix this as in the rest of the PRs?

Reply:
This one actually makes sense I think; `gulp` is a command-line build tool, and isn't useful without `node`.